qbft/spectest/tests/prepare/prepare_quorum_triggered_twice_late_commit.go

```diff
@@ -0,0 +1,39 @@
+package prepare
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// PrepareQuorumTriggeredTwiceLateCommit is a spec test that checks the case where all prepares are sent and quorum event is triggered more than once.
+// A commit message was seen only after the last prepare
+func PrepareQuorumTriggeredTwiceLateCommit() tests.SpecTest {
```

Comment:
why is this a special case?

Reply:
See #214 for a bug on this case.
We send two commit messages instead of one.
After consulting @lior-blox and @y0sher it was  decided it is low priority for now.
This is because networking cache takes care of this issue, and even if it didn't it is unclear how it can be a problem (if we disregard message validation).

Reply:
@GalRogozinski it is to be discussed what do we want to do when we get a message of type that we already get a quorum of. 
This acts the same on prepare and also on commits, I was sure this is a design decision. but maybe we can stop processing after quorum? or at least not broadcast another message?

Reply:
@y0sher Maybe,
The problem is that you don't know if you broadcasted the first commit message after you receive it from gossip